lib/common/strings.c

```diff
@@ -665,7 +665,7 @@ copy_str_table_entry(gpointer key, gpointer value, gpointer user_data)
  * \internal
  * \brief Copy a hash table that uses dynamically allocated strings
  *
- * \param[in] old_table  Hash table to duplicate
+ * \param[in,out] old_table  Hash table to duplicate
```

Comment:
Do we modify `old_table` or anything in it? Looks like we just `strdup` the keys and values